docs/analyzer/user_guide.md

```diff
@@ -445,6 +445,10 @@ my/gcc/compiler/g++ main.cpp  # Not captured because there is no match.
 my/gcc/compiler/gcc-7 main.c  # Captured because "gcc" is infix of "gcc-7".
 /usr/bin/g++ main.cpp         # Captured because "/bin/g++" is postfix of the compiler path.
 /usr/bin/g++-7 main.cpp       # Not captured because "/bin/g++" is not postfix of the compiler path.
+
+# For an exact compiler binary name match start the binary name with a "/".
+/clang # will not log calls to clang-tidy only the clang binary calls will be captured
```

Comment:
Instead of `clang-tidy` we have to mention `clang++`. Why the user would like to log `clang-tidy` calls?

Also please start the comment (sentence) with capital letter and write a punctuation mark to the end.

Reply:
yes the example is here to highlight that it might log extra binaries which are not wanted, I've extended the comment